apidef/oas/server.go

```diff
@@ -58,3 +60,20 @@ func (lp *ListenPath) ExtractTo(api *apidef.APIDefinition) {
 	api.Proxy.ListenPath = lp.Value
 	api.Proxy.StripListenPath = lp.Strip
 }
+
+type ClientCertificates struct {
+	// AllowList is the list of client certificates which are allowed.
+	AllowList []string `bson:"allow_list" json:"allow_list"`
```

Comment:
in the contract, it is `allowlist`. The parameter name should be `Allowlist` according to that.